Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM node:14-alpine as node-build
```

Comment:
I would recommend splitting this in two docker files. Or use stages such that you have a development stage which still includes all the tools such as npm and composer.

Reply:
I think that the idea of the current setup is that is should be faster. I was thinking about making a docker-compose override file for development. In this I could just re-add some stuff that is useful.